Sample_v3/Samples/SwiftUISimpleChat/ChannelListView.swift

```diff
@@ -203,6 +210,20 @@ struct ChannelListView: View {
             ).observableObject
         )
     }
+    
+    private func createDirectMessageChannel(with userId: UserId, completion: @escaping () -> Void) {
+        let client = channelList.controller.client
+        
+        let newChannelController = try! client.channelController(
+            createDirectMessageChannelWith: [client.currentUserId, userId],
+            extraData: .init()
+        )
+        
+        newChannelController.synchronize { [newChannelController] _ in
+            _ = newChannelController
+            completion()
+        }
```

Comment:
I don't understand this. Why we create a controller just to create a channel and then throw it away?